tests/conftest.py

```diff
@@ -549,7 +549,7 @@ def _create_aggregated_surface_dataset(rmsglobalconfig, regsurf, content):
     for i in range(10):  # TODO! 10
         use_regsurf = regsurf.copy()
         use_regsurf.values += float(i)
-        expfile = edata.export(use_regsurf, name="mymap_" + str(i), realization=i)
+        expfile = edata.export(use_regsurf, name="mymap_" + str(i))
```

Comment:
`realization` argument is deprecated and has no effect

Reply:
Maybe consider `name=f"mymap_{i}"` while we're here